maven/bnd-maven-plugin/README.md

```diff
@@ -13,6 +13,25 @@ All of the above artifacts will be generated into the project build output
 directory, i.e. `target/classes`, to be subsequently picked up and included
 in the JAR file by the default `maven-jar-plugin`.
 
+The plugin only provides one goal `bnd-process` which is doing that. 
+It is not executed by default, therefore at least one explicit execution needs to be configured (by default bound to phase `process-classes`)
+
+```xml
+<plugin>
+    <groupId>biz.aQute.bnd</groupId>
+    <artifactId>bnd-maven-plugin</artifactId>
+    <version>3.3.0</version>
```

Comment:
We probably ought not include a specific version in the readme since the version of the plugin will change each release and we will never remember to update this line.